app/models/ability.rb

```diff
@@ -29,5 +29,15 @@ def initialize(user)
     else
       can :view, :all
     end
+
+    #check if user's chapter is active
+    #if not, only let them see Dashboard with information about deactivation
+    @user_chapter = Chapter.where(id: user.chapter_id)
+    @user_chapter.each do |c|
+      if c.is_active != 1
```

Comment:
Changing this field to a boolean would make this pretty clean. It could just be:

```
if !c.is_active
```

Or something along these lines.